examples/everything.amp.html

```diff
@@ -318,6 +341,14 @@ <h1 id="top">AMP #0</h1>
   <div id="footer">The end. <a href="#top">Back to top.</a></div>
 
   <amp-pixel src="https://pubads.g.doubleclick.net/activity;dc_iu=/12344/pixel;ord=$RANDOM?"></amp-pixel>
+  <amp-font
+    layout="nodisplay"
+    font-family="Source Sans Pro"
+    timeout="3000"
+    while-loading-class="sanspro-font-loading"
```

Comment:
What is the `while-loading-class` for? The problem is that this class won't be set until amp-font library is actually loaded. Another issue is that this usecase is easily implementable with the API suggested here #648. So we should err on a side of making API smaller.